.env_sample

```diff
@@ -40,19 +41,27 @@ OPENRESTY_URL=""
 FORCE_SSL_IN_RAILS="not applicable in dev"
 
 # Postgres DB
-DATABASE_NAME="Forem_development" # Used when creating a db instance `rails db:create`
-DATABASE_URL="postgresql://localhost:5432/$DATABASE_NAME" # This takes precedence over `DATABASE_NAME` if they don't match
-RAILS_MAX_THREADS=5 # Specifies the number of Puma threads
-WEB_CONCURRENCY=2 # Specifies the number of `workers` to boot in clustered mode.
-DATABASE_POOL_SIZE=5 # The number of concurrent connections to Postgres. If unset the value of RAILS_MAX_THREADS will be used in it's place.
+# Used when creating a db instance `rails db:create`
+DATABASE_NAME="Forem_development"
+# This takes precedence over `DATABASE_NAME` if they don't match
+DATABASE_URL="postgresql://localhost:5432/$DATABASE_NAME"
```

Comment:
The variable $DATABASE_NAME should be substituted with the real value -> `Forem_development`. Same for the test db URL. Got an error too.

Reply:
Mmm that's weird, I just tried with `foreman` and the variable substitution works

Reply:
Variable didn't work when building in wsl

Reply:
Ah! you're right! It works in the console, but it doesn't with Foreman